make_deb

```diff
@@ -22,73 +22,60 @@
 set -e
 
 GIT_ROOT=`pwd`    # The script must be executed from the root git directory
-architecture="none"
-usage="$(basename "$0") {arm} [clean|cleanall]
+usage="$(basename "$0") help [clean|cleanall]
 This script is used to create the Debian package of foglamp dht11 plugin
 
 Arguments:
- arm      - Build an armv7l package
+ help     - Display this help text
  clean    - Remove all the old versions saved in format .XXXX
  cleanall - Remove all the versions, including the last one"
 
-if [ $# -gt 0 ]
-then
-    for i in "$@"
-    do
-      case "$i" in
-          arm)
-              architecture="armhf"
-              shift
-              ;;
-          clean)
-              echo -n "Cleaning the build folder from older versions..."
-              find "${GIT_ROOT}/packages/Debian/build" -maxdepth 1 | grep '.*\.[0-9][0-9][0-9][0-9]' | xargs rm -rf
-              echo "Done."
-              shift
-              ;;
-          cleanall)
-              if [ -d "${GIT_ROOT}/packages/Debian/build" ]; then
-                echo -n "Cleaning the build folder..."
-                rm -rf ${GIT_ROOT}/packages/Debian/build/*
-                echo "Done."
-              else
-                echo "No build folder, skipping cleanall"
-              fi
-              shift
-              ;;
-          help)
-              echo "${usage}"
-              exit 1
-              ;;
-          *)
-            echo "Unrecognized option: $i"
-            exit 1
-            ;;
-      esac
-    done
-else
-  echo "See help and select the architecture to use, *x86* or *arm*"
+if [ "$(dpkg --print-architecture)" != "armhf" ]; then
```

Comment:
Is it okay to use? Or with `uname -m`?

Reply:
I think this is fine as this package only works with ARM because of the GPIO pins

Reply:
I would suggest ` if [ "$(dpkg --print-architecture)" != "arm*" ]; then ` to keep it generic for ARM